app/src/main/java/com/duckduckgo/app/browser/defaultbrowsing/prompts/DefaultBrowserPromptsExperimentImpl.kt

```diff
@@ -421,6 +430,13 @@ class DefaultBrowserPromptsExperimentImpl @Inject constructor(
         pixel.fire(it.pixelName, it.params)
     }
 
+    @Throws(NumberFormatException::class)
+    private fun FeatureSettingsConfigModel.toFeatureSettings() = FeatureSettings(
+        activeDaysUntilStage1 = activeDaysUntilStage1.toInt(),
+        activeDaysUntilStage2 = activeDaysUntilStage2.toInt(),
+        activeDaysUntilStop = activeDaysUntilStop.toInt(),
+    )
+
```

Comment:
Follow up to the comment above, why do we want to do this when we can just have an `Int` in the config and parse it directly to the appropriate type?